src/main.ts

```diff
@@ -21,6 +21,9 @@ async function run(): Promise<void> {
     const repo = check.reposFile() || check.githubRepository()
     const user = await github.getAuthUser(token)
 
+    const authorEmail = core.getInput('author-email') ?? user.email
+    const authorName = core.getInput('author-name') ?? user.name
```

Comment:
In this case, we should use `||` instead of `??`. `||` checks for `0`, `undefined`, `false`, `''`, `null`, `NaN` and `0n`, while `??` only checks for `undefined` or `null` (a missing input [will return `''`](https://github.com/actions/toolkit/blob/9ba7c679ad74f6a80017feeb94b7a7c19b590796/packages/core/src/core.ts#L71)).

```suggestion
    const authorEmail = core.getInput('author-email') || user.email
    const authorName = core.getInput('author-name') || user.name
```